src/pages/Ticket/Seat/SeatInfo/BtnSeat/BtnSeat.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import Common from '../../../../../components/Common/Common';
+
+const BtnSeatSelect = props => {
+  const { ScreenOut, IcoMovie } = Common;
+  const { expanded, expandButton, isDisabled, backgroundPosition } = props;
```

Comment:
```suggestion
const BtnSeatSelect = ({ expanded, expandButton, isDisabled, backgroundPosition }) => {
  const { ScreenOut, IcoMovie } = Common;
```

구조분해 할당을 인자에서 진행할 수 있습니다!

Reply:
수정했습니다..!